### ISSUE TITLE
Relics Balance

### ISSUE THREAD

**Issue by Snoopowo** [EnigmaticaModpacks/Enigmatica10#93]
### Describe your suggestion

- Reduce the overhaul Relic exp needed to level them up
- Change how some of the abilities scale with leveling them up or quality, and adjusting base values to go along with it
- Maybe change some of the loot chance for some of them? Allowing them to spawn in specific modded structures (or trial chambers maybe?), and upping the chance for some that spawn in rarer structures or structures with a low amount of chests.

### Why would you like this added/changed?

- Leveling most of the relics is a pain. Some of them require to litterally just spam right click for up to an hour straight, or just waiting for the cooldown to reset just to use them again. I also think the vanilla XP cost for rerolling / leveling them up is absurdly high, as in older versions of the mods it required XP points and not entire levels.

- The scaling is completely whack. This goes mostly for Relics with cooldowns, since for multiple of them it's part of the upgradeable stats. I'm gonna use the Shadow Glaive (throwable homing weapon, which bounces between enemies and has 8 charges) as an example. At  level 1, the main ability has a recharge time of 30 seconds at lowest quality, and 10 seconds at its highest. At max level it goes from 3s at the lowest and 1s at the highest. Please note the damage, amount of bounces and bounce range are also upgraded along with the cooldown. 

- Looking at the config and the wiki (https://github.com/SSKirillSS/relics/wiki/Configs), it looks like having a stat be fixed would be pretty easy. Putting the initial value to be somewhere between 8 and 5 seconds for example, and the upgrade modifier to be additive with a value of 0 would mean the value doesn't increase with levels. This would probably need to be done on a case by case for all Relics, though.

- The config files allow you to dictate where the Relics can spawn. This makes it really easy to tweak where they can be found. I don't think I've seen any Artifact or Relic appear in Trial Chambers? On top of that, some of them spawn in rare structures / structures with almost no chests, making them very rare for no reasonable reason.

**Comment by NielsPilgaard** [comment]
Hi there, thanks for the suggestion.
I think this sounds reasonable, but I haven't played enough around with Relics to know whether I agree or not 😅

**Comment by MuteTiefling** [comment]
Yeah the one I played around with (infinite ham) was taking so long to level up that I likely wouldn't have seen the level up ability for a few weeks.

They'll all need individual attention, but generally speeding up leveling shouldn't be too game breaking.

**Comment by MuteTiefling** [comment]
Just saw on their discord that a big rework is coming though. Dunno if that's soon or Soon(tm), however. So may or may not be worth putting the time on this just yet.

**Comment by NielsPilgaard** [comment]
Alrighty, let's park this for a week or two 👍
